src/services/config-notification.js

```diff
@@ -29,3 +29,8 @@ export function connectNotificationsWsUpdateConfig() {
     };
     return reconnectingWebSocket;
 }
+
+export function getAnnouncementsWsUrl() {
+    const webSocketBaseUrl = getWsBase();
+    return webSocketBaseUrl + PREFIX_CONFIG_NOTIFICATION_WS + '/global';
```

Comment:
Could be inlined

Reply:
I know I just kept the same style as above
https://github.com/gridsuite/gridstudy-app/blob/7112ee75c9faae09e23d3194be899615ef10145c/src/services/config-notification.js#L15-L20